packages/server/package.json

```diff
@@ -11,7 +11,13 @@
     "test:coverage": "jest --coverage",
     "start:dev": "ts-node-dev --respawn --transpileOnly ./src/server.ts",
     "build": "tsc",
-    "build:serve": "tsc && node ./dist/server.js"
+    "build:serve": "tsc && node ./dist/server.js",
+    "migration:generate:linux": "ts-node ./node_modules/.bin/typeorm migration:generate -n migration",
```

Comment:
Will this work for MacOS as well?

Reply:
Yeah, running a script like this seems weird, is there a specific reason?